dotnet/dynamodb/UpdateItemDataModelTest/UpdateItemDataModelTest.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX - License - Identifier: Apache - 2.0
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Amazon.DynamoDBv2.DataModel;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace DynamoDBCRUD
+{
+    public class UpdateItemDataModelTest
+    {
+        private readonly string _endpointURL = "http://localhost:8000";
+        private readonly string _tableName = "CustomersOrdersProducts";
+        private static readonly string _id = "16";
+        private readonly string _keys = "Area,Order_ID,Order_Customer,Order_Product,Order_Date,Order_Status";
+        private readonly string _values = "Order,11,5,4,2020-05-11 12:00:00,delivered";
+        private readonly string _status = "pending";
+        private readonly ITestOutputHelper output;
+
+        public UpdateItemDataModelTest(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        private IDynamoDBContext CreateMockDynamoDBContext(AmazonDynamoDBClient client)
+        {
+           
+            var mockDynamoDBContext = new DynamoDBContext(client);
+
+            return mockDynamoDBContext;
+        }
+
+        public static async Task<CreateTableResponse> MakeTableAsync(IAmazonDynamoDB client, string table)
+        {
+            var response = await client.CreateTableAsync(new CreateTableRequest
+            {
+                TableName = table,
+                AttributeDefinitions = new List<AttributeDefinition>
+                {
+                    new AttributeDefinition
+                    {
+                        AttributeName = "ID",
+                        AttributeType = "S"
+                    },
+                    new AttributeDefinition
+                    {
+                        AttributeName = "Area",
+                        AttributeType = "S"
+                    }
+                },
+                KeySchema = new List<KeySchemaElement>
+                {
+                    new KeySchemaElement
+                    {
+                        AttributeName = "ID",
+                        KeyType = "HASH"
+                    },
+                    new KeySchemaElement
+                    {
+                        AttributeName = "Area",
+                        KeyType = "RANGE"
+                    }
+                },
+                ProvisionedThroughput = new ProvisionedThroughput
+                {
+                    ReadCapacityUnits = 10,
+                    WriteCapacityUnits = 5
+                }
+            });
+
+            return response;
+        }
+       
+        public static async Task<bool> AddItemAsync(IAmazonDynamoDB client, string table, string id, string keystring, string valuestring)
+        {
+            // Get individual keys and values
```

Comment:
Missing punctuation, here and throughout.

Reply:
Fixed.